examples/checkout/index.js

```diff
@@ -0,0 +1,91 @@
+$(function() {
+
+  /* Build new ShopifyBuy client
+  ============================================================ */
+  var client = ShopifyBuy.buildClient({
+    apiKey: 'bf081e860bc9dc1ce0654fdfbc20892d',
+    myShopifyDomain: 'embeds',
+    appId: '6'
+  });
+
+  var product;
+  var cart;
+
+  var cartPromise = client.createCart();
+  var productPromise = client.fetchProduct('3614436099');
+  RSVP.all([cartPromise, productPromise]).then(function (values) {
```

Comment:
Yeah, try to be polyfill agnostic. I'd like to remove RSVP and run on core-js eventually.